leaf-ui/scripts/main.js

```diff
@@ -26,7 +28,7 @@ var reader;
 //Properties for both core and simulator.
 //TODO: merge this two arrays in order to make use the same name for all
 var satvalues = {
-		"satisfied": "2", "partiallysatisfied": "1", "partiallydenied": "-1", "denied": "-2", "unknown": "4", "conflict": "3", "none": "0",
+		"satisfied": 2, "partiallysatisfied": 1, "partiallydenied": -1, "denied": -2, "unknown": 4, "conflict":3, "none": 0,
```

Comment:
We might need two of these. One with quotes and one without. @g2hchang added the quotes to fix a different bug.